src/wast-parser.y

```diff
@@ -1531,14 +1500,14 @@ script :
 
       int last_module_index = -1;
       for (size_t i = 0; i < $$->commands.size(); ++i) {
-        Command& command = *$$->commands[i].get();
+        Command* command = $$->commands[i].get();
```

Comment:
Haha, definitely! It's pretty silly that every caller has to manually manage the module this way. It would be a nice cleanup for someone to do. Or likely I'll get around to it in a few PRs. :-)